src/closedwindows/closedcard-controller.js

```diff
@@ -0,0 +1,13 @@
+(function() {
+    'use strict';
+
+    class ClosedcardCtrl {
+        constructor() {
+
+        }
+    }
+    ClosedcardCtrl.$inject = [];
+
+    angular.module('openfin.closedcard', [])
```

Comment:
I don't understand. I thought this follows the pattern of every other controller.